jujugui/static/gui/src/app/components/deployment-flow/sshkey/test-sshkey.js

```diff
@@ -44,11 +44,16 @@ describe('DeploymentSSHKey', function() {
   });
 
   // Render the component and return the instance and the output.
-  const render = (cloudType, _getGithubSSHKeys) => {
+  const render = (cloudType, _getGithubSSHKeys, user) => {
     let cloud = null;
     if (cloudType) {
       cloud = {cloudType: cloudType};
     }
+    let userInstance = null;
+    if (user) {
+      userInstance = function() {};
```

Comment:
No I really am confused about whether this should be a function or an object :).